.yarnrc

```diff
@@ -1,2 +1,2 @@
 workspaces-experimental true
---install.frozen-lockfile true
```

Comment:
don't think this line should be deleted 🤔

Reply:
oops